tests/runci/targets/Cpp.hx

```diff
@@ -24,7 +24,7 @@ class Cpp {
 			var path = getHaxelibPath("hxcpp");
 			infoMsg('hxcpp has already been installed in $path.');
 		} catch(e:Dynamic) {
-			haxelibInstallGit("HaxeFoundation", "hxcpp", true);
+			haxelibInstallGit("RblSb", "hxcpp", "map_tostring", true);
```

Comment:
please, revert this.

Reply:
Why did you "resolve" this?

Reply:
Might have been automatic. Will need to have it reverted before merging in any case (but yeah it's fine to keep it around for tests on this PR)